src/main/java/uk/gov/hmcts/reform/divorce/orchestration/controller/advice/GlobalExceptionHandler.java

```diff
@@ -59,6 +59,13 @@
             );
     }
 
+    @ExceptionHandler(CaseNotFoundException.class)
+    ResponseEntity<Object> handleCaseNotFoundException(CaseNotFoundException exception) {
+        log.warn(exception.getMessage(), exception);
+
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exception.getMessage());
+    }
```

Comment:
Coverage of this is shown as 0 in sonar but it is actually covered by integration test